app/Models/Location.php

```diff
@@ -127,7 +128,10 @@ protected function fetchCoordinates()
         try {
             /** @var \LibreNMS\Interfaces\Geocoder $api */
             $api = app(\LibreNMS\Interfaces\Geocoder::class);
-            $this->fill($api->getCoordinates($this->location));
+
+            // Removes Location info inside () when looking up lat/lng
+            $this->fill($api->getCoordinates(preg_replace($this->location_ignore_regex, '', $this->location)));
+            // $this->fill($api->getCoordinates($this->location));
```

Comment:
You'll need to remove this commented code, thanks :)

Reply:
removed